cli/cmd/remove.go

```diff
@@ -0,0 +1,47 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/apex/log"
+	"github.com/spf13/cobra"
+	"github.com/tarantool/tt/cli/cmdcontext"
+	"github.com/tarantool/tt/cli/configure"
+	"github.com/tarantool/tt/cli/modules"
+	"github.com/tarantool/tt/cli/remove"
+)
+
+// NewRemoveCmd creates remove command.
+func NewRemoveCmd() *cobra.Command {
+	var removeCmd = &cobra.Command{
+		Use:   "remove what",
+		Short: "Remove tarantool/tt",
+		Run: func(cmd *cobra.Command, args []string) {
+			err := modules.RunCmd(&cmdCtx, cmd.Name(), &modulesInfo, InternalRemoveModule, args)
+			if err != nil {
+				log.Fatalf(err.Error())
+			}
+		},
+	}
+
+	return removeCmd
+}
+
+// InternalRemoveModule is a default remove module.
+func InternalRemoveModule(cmdCtx *cmdcontext.CmdCtx, args []string) error {
+	cliOpts, err := configure.GetCliOpts(cmdCtx.Cli.ConfigPath)
+	if err != nil {
+		return err
+	}
+	if len(args) == 0 {
```

Comment:
Use cobra functionality for this check: OnlyValidArgs
Also, do not print help using log. Let cobra print usage help.